sphinx/source/docs/user_guide/notebook.rst

```diff
@@ -23,19 +23,16 @@ are required. You can see an example below:
     As a convenience, |output_notebook| is also importable from the
     |bokeh.charts| and |bokeh.plotting| modules.
 
-Connecting to Bokeh Server Plots
---------------------------------
-
 
-Integrating IPython Interactors
--------------------------------
+Integrating Notebook Interactors
+--------------------------------
 
-It is possible to drive updates to Bokeh plots using IPython/Jupyter
+It is possible to drive updates to Bokeh plots using Jupyter
 notebook widgets, known as interactors. The key doing this is the
-|push_notebook| method on |ColumnDataSource|. This method allows you to
-update plot data sources in the notebook, so that the plot is made to
-update. Typically, |push_notebook| is used in the update callback for the
-interactor. An example is shown below:
+|push_notebook| function. This function allows you to update document
+data and properties in the notebook, so that any plots, etc are made to
+update. A common use of |push_notebook| is in an update callback for
+interactors. An example is shown below:
```

Comment:
Is there a way to update multiple plots in the notebook?  (this is somewhat important to me)